pkgs/development/python-modules/dash-bootstrap-components/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, dash
+}:
+
+buildPythonPackage rec {
+  pname = "dash-bootstrap-components";
+  version = "1.3.0";
+
+  src = fetchPypi {
+    pname = "dash-bootstrap-components";
+    inherit version;
+    sha256 = "sha256-8S8pn9yRlDxw15UElJNlpiL9nKWihLU1lC31zEecE1Q=";
+  };
+
+  propagatedBuildInputs = [ dash ];
+
+  # No tests in archive
```

Comment:
Please fetch from github, so we can run the tests https://github.com/facultyai/dash-bootstrap-components/tree/main/tests